parse.js

```diff
@@ -6,40 +6,33 @@
 	License: Apache-2.0
 */
 
-const ARGUMENT_SEPARATION_REGEX = /([^=\s]+)=?\s*(.*)/;
+const ARGUMENT_SEPARATION_REGEX = /^([^=\s]+)=?\s*(.*)/;
```

Comment:
Why do we need `^` here?

Reply:
^ makes sure the match is at the beginning. Considering how lax the regex is, there is little difference between using it and not.
One key difference is that "=a=1" would be parsed like "a=1" whereas with the ^, it would not match at all.